packages/graph/hash_graph/tests/integration/postgres/data_type.rs

```diff
@@ -52,14 +56,15 @@ async fn update() {
         .await
         .expect("could not seed database");
 
-    api.create_data_type(&object_dt_v1)
+    api.create_data_type(object_dt_v1)
         .await
         .expect("could not create data type");
 
-    api.update_data_type(&object_dt_v2)
+    api.update_data_type(object_dt_v2)
         .await
         .expect("could not update data type");
 
-    assert_ne!(object_dt_v1, object_dt_v2);
-    assert_ne!(object_dt_v1.id(), object_dt_v2.id());
+    // TODO: what's the point of this check
```

Comment:
I think these were a bug? 

Perhaps the intention was to check the objects returned from these functions, but I think that was lost along the way.

Reply:
These were used to check the returned types from `create_data_type` and `update_data_type`, but as we don't return them anymore the tests are pointless without a `get_data_type`. We can remove them.

Reply:
The functionality is currently also tested in the REST endpoint tests, so there is no strict need. However, putting a `get` here isn't much work, so probably it's still worth it 🙂

Reply:
I added a few more checks but also encapsulated a TODO because I think the checks aren't particularly helpful as is